src/encoded/static/components/inputs/object.js

```diff
@@ -83,7 +87,7 @@ var ObjectPicker = module.exports.ObjectPicker = React.createClass({
         var previewUrl = url;
         var searchUrl = '/search/' + this.state.search;
         var actions = [
-            <button className="btn btn-primary" onClick={this.handleSelect}>Select</button>
+            <button key={1} className="btn btn-primary" onClick={this.handleSelect}>Select</button>
```

Comment:
No — this refers to a React key. Seems very strange usage though. I must have put it in to fix a problem, but not one I currently understand. I’ll try this locally.